script/main.js

```diff
@@ -249,9 +249,22 @@ class App extends Component {
     this.setState({versions: e.target.value.split(',')});
   }
 
+  collapseVersions(versions) {
+    versions.forEach((version, i) => {
+      if (version < 5) {
+        throw new Error(`Unsupported Lighthouse version (${version})`);
+      }
+      if (version < 6) return;
+      // Back to most recent _even_ version. (As of v6, Lighthouse's even versions change scoring, but odd versions don't.)
+      versions[i] = version - (version % 2)
+    });
+  }
+
   render() {
     const {versions, device, metricValues} = this.state;
 
+    this.collapseVersions(versions);
```

Comment:
Does this have to mutate? Can it return a new array and be used on L281?

Reply:
ill let you take another look as i changed things quite a bit.